templates/default/fulldoc/html/css/style.css

```diff
@@ -1,16 +1,18 @@
-html {
+html { margin: 0; padding: 0;
```

Comment:
Yes. The rest of this rule is formatted as a declaration on a separate line. This change seems to break that convention.